crates/pallet-domains/src/block_tree.rs

```diff
@@ -158,9 +158,17 @@ pub(crate) fn verify_execution_receipt<T: Config>(
         ..
     } = execution_receipt;
 
+    // Checking if the incoming ER is expected regarding to its `domain_block_number`/freshness
```

Comment:
what does "`domain_block_number`/freshness" mean in the comment here ?

Reply:
The number of the domain block that derived the ER, compared to the current head receipt number.

Reply:
hmm. I'm confused with `domain_block_number / freshness` like we freshness is a number and this is a division. Do we need `/` between them ?

Reply:
`/` also served as or 🤣

Reply:
explicit `or` or `||` would have been easier to understand the intention here actually